WordPress/create-block-theme
# Try: refactor clone to copy theme to current site

This PR is an attempt to address #284 #288 and #186, by modifying the behavior of the Clone option.

Instead of generating a zip, _Clone_ creates a copy of the currently active theme, along with any user modifications, to the current site — providing a convenient way for users to effectively fork an existing theme. 

_Export_ was updated to work like Clone did — allowing users to edit the metadata for an export.

**notes**
- The PR does not activate the theme — that still needs to be done manually.
- If we like this idea, I think we should add the ability to edit the metadata to the Export option as a fast follow up, which would solve #186 EDIT this PR was refactored to solve #186 also.

**To test**
- Make some changes to templates and global styles via the site editor
- Use the Clone option
- Verify the cloned theme appears as expected in Appearance > Themes
- Verify that the export and overwrite options still work as expected
